### PR TITLE
Fix: Benchmark errors

### LINKED ISSUE #2: Error in benchmark for native omit
## Expected behaviour
Benchmark for `_.omit()` omits two fields `age` and `active` from the object. Benchmark for native alternative should look like `const { age, active, ...res } = users[1];`.

## Actual behaviour
Benchmark for native code looks for non-existent fields `a`, `c`: `const { a, c, ...res } = users[1];`. Therefore two benchmarks cannot be compared.

## How to reproduce

https://jsfiddle.net/2fjyLuc6/

## Possible solution(s)

Fix the benchmarks
- https://github.com/Berkmann18/NativeVsUtils/blob/c4ad4175375eb0ba345c03536dc8d860de2626ff/index.js#L406-L412
- https://github.com/Berkmann18/NativeVsUtils/blob/c4ad4175375eb0ba345c03536dc8d860de2626ff/nativeVsLodash.js#L365-L370
- https://github.com/Berkmann18/NativeVsUtils/blob/c4ad4175375eb0ba345c03536dc8d860de2626ff/lodash.js#L197

## Environment
-   Version: 
-   OS(s): 
-   Browser(s) (if applicable): 
-   Node version: 
-   NPM version:

### PR DESCRIPTION
**Pull request by victor-homyakov**
## Checklist

-   [x] The PR follows the [guidelines](CONTRIBUTING.md)
-   [x] Passes all the tests (including added/updated ones)
-   [x] Contains a valid documentation
-   [x] It's not a duplicate from another PR

## Changes
This PR implements:
-   [x] Bugfix
-   [ ] Feature
-   [ ] Refactoring
-   [ ] Documentation/wiki/tutorial update
-   [ ] Code style update
-   [ ] Build related change
-   [ ] Other, please specify:

## Details
What does this PR implements exactly?
- removed references to `__`, because there is no Underscore (`__` is undefined)
- fixed incorrect invocation of native `reduce` and `reduceRight`
- fixed missing local variable `x`
- fixed incorrect invocation of `_.replace`

What issues does this resolve?

If it completes one or more tasks (listed in [TODO](TODO.md)), which ones are those?

### CODE REVIEW

File: nativeVsLodash.js [hunk 1]
```diff
@@ -109,9 +109,6 @@ const a0 = [1];
 concatSuite.add('lodash', () => _.concat(a0, 2, [3], [
     [4]
   ]))
-  .add('underscore', () => __.concat(a0, 2, [3], [
```
Comment (Berkmann18) [inline nativeVsLodash.js:112]: It's true that there weren't any definition of `__` which should have been the case, so this doesn't need to be removed.
Edit: nevermind, just realised that was the one with Underscore excluded.

File: nativeVsLodash.js [hunk 2]
```diff
@@ -247,7 +240,7 @@ const redux = (prev, crt) => prev + crt;
 
 reduceSuite
   .add('lodash', () => _.reduce(a1, redux))
-  .add('native', () => arr.reduce(a1, redux))
+  .add('native', () => a1.reduce(redux))
```
Comment (Berkmann18) [inline nativeVsLodash.js:243]: Thanks for catching that and the rest 😁.

**Review by Berkmann18 — CHANGES_REQUESTED** [review]

**Review by Berkmann18 — APPROVED** [review]

**Comment by Berkmann18** [comment]
Would it be okay if I add you to the All Contributors table?

**Comment by victor-homyakov** [comment]
> Would it be okay if I add you to the All Contributors table?

OK

**Pull request merged**